evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -39,15 +39,21 @@
 {% endif %}
 
 {% if evaluation.num_textanswers == 0 %}
-    <td class="icon" data-col="state-textanswers" data-order="2" data-filter="no_textanswers">
+    <td class="icon" data-col="state-textanswers" data-order="3" data-filter="no_textanswers">
         <span class="fas fa-comment icon-gray" data-toggle="tooltip" data-placement="top" title="{% trans 'No text answers available' %}"></span>
     </td>
 {% elif evaluation.num_textanswers > evaluation.num_reviewed_textanswers %}
-    <td class="icon" data-col="state-textanswers" data-order="1" data-filter="unreviewed_textanswers">
-        <span class="fas fa-comment icon-yellow" data-toggle="tooltip" data-placement="top" title="{% trans 'Text answers awaiting review' %}"></span>
-    </td>
+    {% if evaluation.course.final_grade_documents or evaluation.course.gets_no_grade_documents or not evaluation.course.is_graded %}
+        <td class="icon" data-col="state-textanswers" data-order="1" data-filter="unreviewed_textanswers_urgent">
+            <span class="fas fa-comment icon-red" data-toggle="tooltip" data-placement="top" title="{% trans 'Text answers awaiting urgent review' %}"></span>
```

Comment:
What about “Text answers awaiting urgent review as grading process was already completed”?
(Similar for the urgent publication state)

Reply:
unfortunately, the case that the course is ungraded would still be missing in this message. any further ideas?

Reply:
I am not quite satisfied with these, but maybe they give you some other inspirations:

“Text answers awaiting urgent review because necessary grade documents were uploaded”
“Results not yet published although all necessary grade documents were uploaded”

“Text answers awaiting urgent review because no missing grade document hinders publication”
“Results not yet published although no grade document is missing”

(but of course, if you think the string is fine, we can keep it)

Reply:
i will go with "Text answers awaiting urgent review because grading process is finished" as we can argue that if a course is ungraded, its grading process is immediately finished.